Documentation/README.md

```diff
@@ -1,20 +1,99 @@
 # Resources
-
 See the following resources to get started.
 
-## .NET Core
+## System.Device.* APIs
 
-* [.NET Core documentation](https://docs.microsoft.com/dotnet/)
-* [Install .NET Core on Raspberry Pi](https://github.com/dotnet/core/blob/master/samples/RaspberryPiInstructions.md)
+* [Device Bindings](https://github.com/dotnet/iot/tree/master/src/devices) - Includes a collection of APIs representing a range of sensors, displays and human interface devices based on System.Device.* APIs.
+* [DevicesApiTester CLI](https://github.com/dotnet/iot/tree/master/tools/DevicesApiTester) - Helpful utility, based on System.Device.* APIs, that include various commands for testing connected development boards and external hardware.
+
+## .NET Core
+* [.NET Core Documentation](https://docs.microsoft.com/dotnet/)
```

Comment:
Although I like the way you separated the content here, I think we probably still want to keep the instructions on how to install .NET Core on a Pi near the top of the document for discoverability. What do you think?